docusaurus.config.js

```diff
@@ -17,77 +17,72 @@ const config = {
+        redirects: [],
+      },
+    ],
+    require.resolve('docusaurus-plugin-hotjar'), 
   ],
 
   presets: [
     [
-      "classic",
-      /** @type {import('@docusaurus/preset-classic').Options} */
-      ({
+      'classic',
+      {
         docs: {
-          sidebarPath: require.resolve("./sidebars.js"),
-          routeBasePath: "/",
+          sidebarPath: require.resolve('./sidebars.js'),
+          routeBasePath: '/',
           // Please change this to your repo.
-          editUrl:
-            "https://github.com/calimero-is-near/docs/edit/master",
+          editUrl: 'https://github.com/calimero-is-near/docs/edit/master',
         },
         theme: {
-          customCss: require.resolve("./src/css/custom.css"),
+          customCss: require.resolve('./src/css/custom.css'),
         },
         gtag: {
           trackingID: 'G-Z9T67N8YSF',
           anonymizeIP: true,
         },
-      }),
+      },
     ],
   ],
 
-  themeConfig:
-    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
-    ({
-      colorMode: {
-        respectPrefersColorScheme: false,
-        disableSwitch: false,
-        defaultMode: 'dark',
-      },
-      metadata: [
-        {name:  'calimero', 
+  themeConfig: {
+    colorMode: {
+      respectPrefersColorScheme: false,
+      disableSwitch: false,
+      defaultMode: 'dark',
+    },
+    metadata: [
+      {
+        name: 'calimero',
         image: 'img/docs_og.jpg',
-        
-      }],
-      image: 'img/docs_og.jpg',
-      navbar: {
-        logo: {
-          alt: "Calimero Logo",
-          src: "img/logo1.svg",
-          srcDark: 'img/logo_dark.svg',
-        },
-        items: [],
-      },
-      algolia: {
-        appId: 'E4LW6HBI8N',
-        apiKey: '659d02800901a93e6214876d98883225',
-        indexName: 'dev_cal'
       },
-      footer: {
-        style: "dark",
-        links: [
-          {
-          },
-        ],
-        copyright: `Copyright © ${new Date().getFullYear()} Calimero Limited.`,
-      },
-      prism: {
-        theme: require('prism-react-renderer/themes/dracula'),
-        additionalLanguages: [
-          "rust", "java", "python", "ruby", "go", "toml"
-        ]
+    ],
+    image: 'img/docs_og.jpg',
+    navbar: {
+      logo: {
+        alt: 'Calimero Logo',
+        src: 'img/logo1.svg',
+        srcDark: 'img/logo_dark.svg',
       },
-    }),
+      items: [],
+    },
+    algolia: {
+      appId: 'E4LW6HBI8N',
+      apiKey: '659d02800901a93e6214876d98883225',
+      indexName: 'dev_cal',
+    },
+    footer: {
+      style: 'dark',
+      links: [
+        // Add your footer links here
+      ],
+      copyright: `Copyright © ${new Date().getFullYear()} Calimero Limited.`,
+    },
+    prism: {
+      theme: require('prism-react-renderer/themes/dracula'),
+      additionalLanguages: ['rust', 'java', 'python', 'ruby', 'go', 'toml'],
+    },
+    hotjar: {
+      applicationId: '3582573'// Replace this with your actual Hotjar Site ID
```

Comment:
is this real id ? Can you also show how it works, post video or screenshots as I dont have access to hotjar account